Assets/Scripts/AI/RacerManager.cs

```diff
@@ -100,5 +112,20 @@ > Vector3.Distance(positionList[i].transform.position, consideredPathNode))
         positionList.Add(racerToAdd);
     }
 
+    public static Rigidbody GetRigidbody(int index) { return racerRigidbodies[index]; }
+
+    public static PositionTracker GetPositionTracker(int index) { return positionTrackers[index]; }
+
+    public static void SetStartTime()
```

Comment:
Name of the function seems to indicate that it is a setter function but it is not. Maybe change the name to something like "RetrieveStartTime"